src/Altinn.App.Core/Models/DataElementChanges.cs

```diff
@@ -122,8 +122,10 @@ public sealed class FormDataChange : DataElementChange
     /// The binary representation (for storage) of the data element after changes
     /// </summary>
     /// <remarks>
-    /// Not available for form data in data processing phase, because it can't reflect
+    /// null during the form data in data processing phase, because it can't reflect
```

Comment:
What happened to this part of the comment?

Reply:
I will fix (and include the tests) soon.

I ran into an issue overriding environment so that this code runs under webapplicationfactory